platform/security/handler/security-handler-api/pom.xml

```diff
@@ -109,17 +109,17 @@
                                         <limit>
                                             <counter>INSTRUCTION</counter>
                                             <value>COVEREDRATIO</value>
-                                            <minimum>0.39</minimum>
```

Comment:
❓  These numbers are awfully close which almost makes me worried about slight environmental differences.

Reply:
Sounds likely to me.